xml/System.Text.Unicode/UnicodeRanges.xml

```diff
@@ -13,8 +13,16 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Provides static methods that return predefined <see cref="System.Text.Unicode.UnicodeRange" /> instances that correspond to blocks from the Unicode 7.0 specification.</summary>
```

Comment:
>7.0  [](start = 161, length = 4)

I am not sure if we need to specify the Unicode version here. we can just omit the version to have more flexibility and don't have to change the documentation later.